common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -370,14 +370,10 @@ bool ap_wlan_hal_dwpal::sta_allow(const std::string &mac)
     return true;
 }
 
-bool ap_wlan_hal_dwpal::sta_deny(const std::string &mac, int reject_sta)
+bool ap_wlan_hal_dwpal::sta_deny(const std::string &mac)
 {
     // Build command string
-    std::string cmd = "DENY_MAC " + mac;
-
-    if (reject_sta) {
-        cmd += " reject_sta=" + std::to_string(reject_sta);
-    }
+    std::string cmd = "DENY_MAC " + mac + " reject_sta=33";
```

Comment:
@morantr Can you confirm that it's OK to always use 33 here?

Reply:
On the client steering task, you removed the following.
```cpp
        /*
        * Only for the original hostap, add error code "33" to signal hostap that we want to continue 
        * to hear BTM response from client.
        */
        vs_tlv->reject_sta() = (hostap == original_radio_mac) ? 33 : 0;
```
So previous we set the reject code to 33 only on the radio the client was connected to, and now we set it to all hostaps. I gotta say I quite don't understand the comment nither the what the standard says about it. 😕

Reply:
Not 100% sure, but I vaguely remember that disassociating a station with a non 33 reason code, in some cases caused an Authentication Error on the STA (Android? Samsung?).
When this happened, the station marked the network (SSID) as invalid and didn't try reconnecting to it at all.
Perhaps it's fine to always use 33 as the reject code.